recipe/meta.yaml

```diff
@@ -41,6 +41,9 @@ requirements:
     - numpy
     - {{ mpi }}  # [mpi != 'nompi']
     - mpi4py  # [mpi != 'nompi']
+    # need to list hdf5 twice to get version pinning from conda_build_config
+    # and build pinning from {{ mpi_prefix }}
+    - hdf5
```

Comment:
@minrk do you think this work-around should be applied to all mpi-varianted packages?
(e.g. `adios`, `openpmd-api` feedstocks)

Reply:
Yeah, I think so